benchmark_scripts/text_generation/run.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2022 The KerasNLP Authors
```

Comment:
Let's move this whole thing into `keras_nlp/benchmarks` maybe. I think having this live next to `integration_tests` makes sense.

Could we also avoid the nested directory? Just a `text_generation.py` file straight in `benchmarks`?